db/schema.rb

```diff
@@ -55,6 +55,33 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "piis", force: :cascade do |t|
+    t.integer  "user_id",                               null: false
+    t.boolean  "active",                default: false, null: false
+    t.boolean  "verified",              default: false, null: false
+    t.datetime "created_at",                            null: false
+    t.datetime "updated_at",                            null: false
+    t.string   "first_name"
+    t.string   "middle_name"
+    t.string   "last_name"
+    t.string   "gen"
+    t.string   "address1"
+    t.string   "address2"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zipcode"
+    t.string   "ssn"
+    t.string   "dob"
+    t.string   "phone"
+    t.string   "drivers_license_state"
+    t.string   "drivers_license_id"
+    t.string   "passport_id"
+    t.string   "military_id"
+  end
```

Comment:
Also, I would recommend dropping gen.  Probably not something we want to be storing  until we really really have to.